pkg/configuration/configuration.go

```diff
@@ -131,30 +137,42 @@ const (
 	DefaultVerificationMessageTemplate = "Your verification code for Red Hat Developer Sandbox is: %s"
 )
 
-// Registry encapsulates the Viper configuration registry which stores the
+// Config encapsulates the Viper configuration registry which stores the
 // configuration data in-memory.
-type Registry struct {
-	v *viper.Viper
+type Config struct {
+	v            *viper.Viper
+	secretValues map[string]string
 }
 
 // CreateEmptyRegistry creates an initial, empty registry.
-func CreateEmptyRegistry() *Registry {
-	c := Registry{
-		v: viper.New(),
+func CreateEmptyRegistry(cl client.Client) (*Config, error) {
```

Comment:
I guess we can rename it now to `CreateEmptyConfig()`

Reply:
done as part of: a2e56a1a11ad6cb36396a0e2afc4d2a0447df3ea